tests/python/test_oop.py

```diff
@@ -222,3 +222,16 @@ def hook(x: ti.template()):
     for i in range(32):
         for j in range(32):
             assert (solver.val[i, j] == 1.0)
+
+
+@ti.test(arch=ti.get_host_arch_list())
+def test_oop_with_portery_decorator():
+    @ti.data_oriented
+    class Test:
+        @property
+        @ti.kernel
+        def test(self) -> ti.i32:
+            return 0
+
+    a = Test()
+    print(a.test)
```

Comment:
```suggestion
    assert a.test == 24
```

Let's make it a real test :-)